gtk/tests_backend/widgets/canvas.py

```diff
@@ -107,7 +107,7 @@ async def alt_mouse_drag(self, x1, y1, x2, y2):
         self.native.emit("button-press-event", event)
 
         event = Gdk.Event.new(Gdk.EventType.MOTION_NOTIFY)
-        event.state = Gdk.ModifierType.BUTTON3_MASK
+        event.state = Gdk.ModifierType.BUTTON3_MASK | Gdk.ModifierType.MOD2_MASK
```

Comment:
As above - a comment for clarity.
```suggestion
        # Add an additional modifier to confirm that keyboard modifiers don't
        # impact event triggering
        event.state = Gdk.ModifierType.BUTTON3_MASK | Gdk.ModifierType.MOD2_MASK
```